src/libs/antares/study/parts/short-term-storage/properties.cpp

```diff
@@ -97,7 +97,7 @@ bool Properties::loadKey(const IniFile::Property* p)
         return p->value.to<std::string>(this->name);
 
     if (p->key == "storagecycle")
-        return p->value.to<unsigned int>(this->cycleDuration);
+        return valueForOptional(this->storagecycle);
 
     if (p->key == "initiallevel")
```

Comment:
For now, if **initialLevel** has no value (for instance **initialLevel  = optim** in the **list.ini**), the code should crash ?